message_definitions/v1.0/common.xml

```
@@ -6711,9 +6717,9 @@
       <field type="uint8_t" name="target_component">Component ID (0 for broadcast).</field>
       <field type="uint8_t[20]" name="id_or_mac">Only used for drone ID data received from other UAs. See detailed description at https://mavlink.io/en/services/opendroneid.html. </field>
       <field type="uint8_t" name="authentication_type" enum="MAV_ODID_AUTH_TYPE">Indicates the type of authentication.</field>
-      <field type="uint8_t" name="data_page">Allowed range is 0 - 4.</field>
-      <field type="uint8_t" name="page_count">This field is only present for page 0. Allowed range is 0 - 5.</field>
-      <field type="uint8_t" name="length" units="bytes">This field is only present for page 0. Total bytes of authentication_data from all data pages. Allowed range is 0 - 109 (17 + 23*4).</field>
+      <field type="uint8_t" name="data_page">Allowed range is 0 - 15.</field>
+      <field type="uint8_t" name="last_page_index">This field is only present for page 0. Allowed range is 0 - 15. See the description of struct ODID_Auth_data at https://github.com/opendroneid/opendroneid-core-c/blob/master/libopendroneid/opendroneid.h.</field>
```

As far as we know there are no implementations of these messages. Just FYI though, there work, changing the name of this field would be breaking - it would mean an implementation written for the old message would silently drop a message sent using this changed format. 

Just something to keep in mind in future.

---

I am aware of that. However, both the name and the interpretation was changed in the updated standard and this will have to follow.
Fortunately all of these messages are still marked as WIP.